src/Types.ts

```diff
@@ -47,11 +47,11 @@ export interface TabsterCoreProps {
      */
     rootDummyInputs?: boolean;
     /**
-     * A callback that will be called for the uncontrolled areas with `trapsFocus`
-     * when Tabster wants to know is the element is currently trapping focus and
-     * Tabster should not interfere with handling Tab.
+     * A callback that will be called for the uncontrolled areas when Tabster wants
+     * to know is the uncontrolled element wants complete control (for example it
+     * is trapping focus) and Tabster should not interfere with handling Tab.
      */
-    checkUncontrolledTrappingFocus?: (element: HTMLElement) => boolean;
+    checkUncontrolledCompletely?: (element: HTMLElement) => boolean;
```

Comment:
This is a breaking API change - If you deprecated this prop and created a new one, it can be released as a minor change

Reply:
I suppose it would be better to keep it like before for now. And I'll rename it in the next PR which will address the callbacks handling as it will also require a major version bump.